rest/data/pod_test.go

```diff
@@ -12,6 +13,15 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func init() {
```

Comment:
I would avoid doing this because it is assuming that we're initializing global state that will never change for testing. Unless there's a _really_ strong reason to do this, we should only set the environment global state in the individual test cases.